Contributing.MD

```diff
@@ -0,0 +1,30 @@
+Contributing to README Template
+
+We welcome your contributions to README Template. This doc describes the process to contribute patches to README Template and the general guidelines we expect contributors to follow.
+Before You Start
+
+We accept patches in the form of github pull requests. If you are new to github, please read How to create github pull requests first.
+Contributor License Agreements
+
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution, this simply gives us permission to use and redistribute your contributions as part of the project.
+If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an individual CLA.
+If you work for a company that wants to allow you to contribute your work, then you'll need to sign a corporate CLA.
+Contributing Process
```

Comment:
Suggest we title this section "Contributing Via Pull Request" or "Submitting a Pull Request"